README.md

```diff
@@ -31,7 +31,7 @@ const { PaperPlaneIcon } = XYZIconSet;
 ```jsx
 import XYZIconSet from 'xyz-icon-set-vue';
 
-const { PaperPlaneIcon } = XYZIconSet;
+import { PaperPlaneIcon } from 'xyz-icon-set-vue';
```

Comment:
The first line is no longer necessary here

Reply:
Solved now